docs/data_managers/overview.md

```diff
@@ -58,3 +58,20 @@ published.
 Once you have installed and configured these tools, then you can use the provided
 [command line tools](command_line_tools/overview.md) to validate and publish datasets.
 The [usage recipes](using_safedata/overview.md) show how the tools are used.
+
+## Example `safedata` format datasets
+
+<!-- The links here are hard coded to main, so if you've changed one of the files but 
+can't see the change in the docs that is why-->
+To help both data managers and data providers understand the `safedata` format we
+provide a number of resources. A template dataset containing the required worksheets,
+labels and headers, can be downloaded
+[here](https://github.com/ImperialCollegeLondon/safedata_validator/blob/main/docs/data_providers/data_format/Template.xlsx).
```

Comment:
I think this link is better as:
```suggestion
[here](https://github.com/ImperialCollegeLondon/safedata_validator/raw/main/docs/data_providers/data_format/Template.xlsx).
```

That will give an actual download rather than the odd GH page view.